### PR TITLE
Added left tree with static values, add js to hide/show left tree, added required CSS for left tree menu. 

### PR DESCRIPTION
**Pull request by divakarkonakalla**
@udeshrl , Please merge the changes.

### CODE REVIEW

File: client/cat3/src/partials/sections/dashboard/analytics/view/analyticsTreeMenu.html [hunk 1]
```diff
@@ -1,3 +1,67 @@
...
+				        </div>
+				    </li>
+				</ul>
+			</div>
+		</div>
+	</div>
+</div>
+
+<script>
+	$(document).ready(function() {
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/analytics/view/analyticsTreeMenu.html]: Remove the this function .

File: client/cat3/src/partials/sections/dashboard/analytics/analytics.js [hunk 2]
```diff
@@ -20,6 +21,24 @@
 		if (!$rootScope.stateParams.view) {
 			$state.go('dashboard.analytics', {menuItem: 'cost', view: 'chat'});
 		}
+
+		$scope.hideTreeOverlay = function () {
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/analytics/analytics.js]: Inside dashboard controller those programs is available.

File: client/cat3/src/partials/sections/dashboard/analytics/analytics.scss [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+
+#analyticsPage {
+  margin-bottom: 20px;
+
+  .no-padding {
+    padding: 0;
+  }
+
+  .headingDashboard {
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/analytics/analytics.scss]: use parent id ( #analyticsPage ).

File: client/cat3/src/partials/sections/dashboard/analytics/view/analyticsTreeMenu.html (hunk 1 above)
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/analytics/view/analyticsTreeMenu.html]: Remove the this function .

**Comment by divakarkonakalla** [comment]
Review comments has been taken care.

@udeshrl , Please merge the changes.

**Pull request merged**